### ISSUE TITLE
“Share” button with an insufficient accessible name [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#216]
The "Share" button label is currently announced by TalkBack as "view or amend application settings". This does not accurately convey the button's "share" functionality. Ensure the button's label conveys its "Share" function, since its purpose is not to change the application settings

![data Tab - Share button](https://user-images.githubusercontent.com/7034772/91988488-4fec7900-ecfd-11ea-9933-712d80846083.png)

Severity: Medium

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed

**Comment by jonnyjames** [comment]
The "Share" button text is now announced by TalkBack as "Share, share this app with friends. Help keep NY safe, button".

I recommend removing the redundant "Share" text and parse it to be "Share this app".

**Comment by floridemai** [comment]
@jonnyjames Applied requested updates

**Comment by jonnyjames** [comment]
@floridemai I am still hearing the extraneous text being announced, will the update be in the next build?

**Comment by floridemai** [comment]
@jonnyjames If it hears "Share this app with friends" then is correct. Otherwise we should have another look at this.

**Comment by jonnyjames** [comment]
This is now fixed. Closing this issue.
